pyanaconda/installation.py

```diff
@@ -68,6 +70,10 @@ def _writeKS(ksdata):
 
     # Make it so only root can read - could have passwords
     with util.open_with_perm(path, "w", 0o600) as f:
+        f.write("# Generated by Anaconda {} with Blivet {}\n".format(
```

Comment:
I think that the Blivet version should be generated by the Storage module. This code is going to live in the Boss module and it shouldn't have to import Blivet because of this string.

Reply:
I'll take a look at this, then...